### PR TITLE
Vmi/upload file

### PR DESCRIPTION
**Pull request by Nikeleos**

### CODE REVIEW

File: src/UploadFile/UploadFile.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+      setUploadButtonDisabled(false);
+    });
+  };
+
+  const handleDownloadFile = () => {
+    const downloadPromise = new Promise((resolve) => {
+      setFileStatus(UPLOAD_FILE_STATUS_KEY.DOWNLOADING);
+      setUploadButtonDisabled(true);
+      downloadFile(fileToDownload.name)
+      // setFileCache(downloadFile(fileName));
```
Comment (COSMO-VMI) [inline src/UploadFile/UploadFile.js:93]: Either set the file cache or remove this comment.

Comment (Nikeleos) [reply src/UploadFile/UploadFile.js:93]: Acutally, if the comment is remove, the feature cannot be tested with success. It's for future binding with workspace.api.

File: src/UploadFile/UploadFile.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+          </Button>
+        </Grid>
+        <Grid item>
+          { fileToDownload.name && fileStatus === UPLOAD_FILE_STATUS_KEY.IDLE &&
+            <Grid container spacing={3} direction="row" justify="flex-start" alignItems="center">
+              <Grid item>
+                <Link component="button" onClick={handleDownloadFile} download>
+                  <Grid container spacing={2} direction="row" justify="flex-start" alignItems="center">
+                    <Grid item>
+                      <GetAppIcon style={{ color: '#FFFFFF', fontSize: 20 }} />
```
[the remark this answers is not in the document]
Comment (Nikeleos) [reply src/UploadFile/UploadFile.js:123]: the style seems to be useless. I remove it.

**Review by jreynard-code — APPROVED** [review]
LGTM

**Pull request merged**